core/common/src/main/java/alluxio/resource/LockResource.java

```diff
@@ -22,7 +23,7 @@
  *   }
  * </pre>
  */
-public class LockResource implements AutoCloseable {
+public class LockResource implements Closeable {
```

Comment:
I'm not sure what the difference is between `AutoCloseable` and `Closeable`. Also, This is still `Closeable`, and `CloseableResource` is now `AutoCloseable`. Should they all be consistent?